seq2science/envs/sctk.yaml

```diff
@@ -0,0 +1,18 @@
+name: sctk
+channels:
+  - bioconda
+  - defaults
+  - conda-forge
+dependencies:
+  - bioconda::bioconductor-singlecelltk=2.4.0
+  - anaconda::scipy=1.8.0
+  - anaconda::astroid=2.9.0
+  - anaconda::six=1.16.0
+  - anaconda::numpy=1.21.5
+  - conda-forge::r-kableextra=1.3.4
+  - pip:
+    - scrublet==0.2.3
+    - scanpy==1.8.2
+    - anndata==0.7.8
+    - bbknn==1.5.1
+    - scanorama==1.7.2
```

Comment:
All of these are on conda. Is there a reason to install them through pip?

Reply:
For some reason, the sctk author use a mix install of conda/pip in their install script [sctk](https://rdrr.io/github/compbiomed/singleCellTK/man/sctkPythonInstallConda.html) and I took that over. I am not sure if it makes a difference, might have to try it out :)